README.md

```diff
@@ -208,8 +208,9 @@ Further details about the coverage test can be found in the
 
 ### Adaptive Coverage
 
-The line coverage is saved in [tests/coverage](tests/coverage). To update the
-coverage before submitting a PR, run the coverage test:
+The line coverage is saved in either [coverage_config_x86_64.json](coverage_config_x86_64.json) or
+coverage_config_aarch64.json, depending on your machine's architecture.
+To update the coverage before submitting a PR, run the coverage test:
```

Comment:
A link can be added to the aarch64 coverage file aswell, and although quite opinionated I think it is better to link to the files using `inline code`.
```suggestion
The line coverage is saved in either [`coverage_config_x86_64.json`](coverage_config_x86_64.json) or
[`coverage_config_aarch64.json`](coverage_config_aarch64.json), depending on your machine's architecture.
To update the coverage before submitting a PR, run the coverage test:
```

Reply:
Thanks for the tip! I didn't know you could combine the link with any styling.